addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java

```diff
@@ -555,6 +556,22 @@ private void setProperties(Device device) {
      *
      * @param device
      */
+    /* (non-Javadoc)
+     * @see org.eclipse.smarthome.core.thing.binding.BaseThingHandler#bridgeStatusChanged(org.eclipse.smarthome.core.thing.ThingStatusInfo)
+     */
+
+    @Override
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        logger.debug("Bridge Status updated to {} for device: {}", bridgeStatusInfo.getStatus().toString(),
+                getThing().getUID().toString());
+        getMaxCubeBridgeHandler();
```

Comment:
Is this line necessary? The handler should always be there and still be the same as during initialize() is called.